pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -371,21 +371,21 @@ effectiveStdenv.mkDerivation {
   ] ++ lib.optionals enableDocs [
     doxygen
     graphviz-nox
-  ] ++ lib.optionals enableCuda (with cudaPackages; [
-    cuda_cudart
-    cuda_cccl # <thrust/*>
-    libnpp # npp.h
-    nvidia-optical-flow-sdk
+  ] ++ lib.optionals enableCuda [
+    cudaPackages.cuda_cudart
+    cudaPackages.cuda_cccl # <thrust/*>
+    cudaPackages.libnpp # npp.h
+    cudaPackages.nvidia-optical-flow-sdk
```

Comment:
@philiptaron nvidia-optical-flow-sdk is not contained in cudaPackages, this breaks opencv builds with cuda enabled.

Reply:
Yes, I messed this up. Fixed in https://github.com/NixOS/nixpkgs/pull/335880.

Reply:
A good example of the pitfalls of `with`!